components/UserList.js

```diff
@@ -1,21 +1,107 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'next/link';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import AccountBoxIcon from '@material-ui/icons/AccountBox';
+import Badge from '@material-ui/core/Badge';
+import { blue as BlueColor } from '@material-ui/core/colors';
 
-const UserList = (props) => {
-  const { users } = props;
+const styless = {
+  root: {
+    flexGrow: 1,
+  },
+  'list-expand': {
```

Comment:
To make the object key name consistent, remove the single quote and using underline(_) instead dash(-), for example `'list-expand'`  to `list_expand`.